manage_breast_screening/mammograms/jinja2/mammograms/record_medical_information.jinja

```diff
@@ -8,7 +8,7 @@
 {% call card({
   "heading": "Symptoms",
 }) %}
-  <p>Any problems or symptoms, including lumps, swelling, rashes or nipple changes</p>
+  <p{% if not presenter.symptom_rows %} class="nhsuk-u-margin-bottom-3"{% endif %}>Any problems or symptoms, including lumps, swelling, rashes or nipple changes</p>
```

Comment:
Can use `nhsukAttributes()` if you like?

Otherwise this looks good to me. Maybe pad the left-hand side for readability?

```suggestion
  <p {%- if not presenter.symptom_rows %} class="nhsuk-u-margin-bottom-3"{% endif %}>Any problems or symptoms, including lumps, swelling, rashes or nipple changes</p>
```